XpringKit/PayID/XRPPayIDClient.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+/// Provides functionality for XRP in the PayID protocol.
+public class XRPPayIDClient: PayIDClient, XRPPayIDClientProtocol {
+
+  /// The XRP Ledger network that this client attaches to.
+  public var xrplNetwork: XRPLNetwork
```

Comment:
This should be a `let` so it is constant.

We call all iVars in a protocol `var` because protocols don't specify the storage mechanism. Note that we've written it as:
```swift
var  xrplNetwork: XRPLNetwork { get }
```

The `{ get }` annotation means we only need to provide a getter (as opposed to `{get set}`). 

Happy to discuss a bit more offline too. Here's the official documentation on the topic: 
https://docs.swift.org/swift-book/LanguageGuide/Protocols.html#ID269

Reply:
thanks for clarification - definitely spent some time figuring on this.  It makes sense now that just because the protocol can declare an iVar with `var` doesn't mean the implementations can't use `let`